python2/koans/about_comprehension.py

```diff
@@ -40,13 +40,13 @@ def test_double_list_comprehension(self):
         comprehension = [ '{0} and {1}'.format(egg, meat) for egg in list_of_eggs for meat in list_of_meats]
```

Comment:
Q: How do multiple 'for'/'if' clauses in a  single comprehension nest? In other words, does the order matter? does the order make any change in this particular example?